### ISSUE TITLE
How do you read audio levels?

### ISSUE THREAD

**Issue by Xeoncross** [cryptix/wav#3]
I am wondering about reading audio levels (every 50ms-100ms). I see the sample [plotWav](https://github.com/cryptix/wav/blob/master/examples/plotWav/main.go#L52-L62) but it looks like the return value is an integer and I'm not sure what the integer represetns since I'm new to the `wav` format.

I thought perhaps the [GetDumbReader()](https://github.com/cryptix/wav/blob/master/reader.go#L194) might be what I need to iterate the track's audio levels over time.

**Comment by cryptix** [comment]
Hey,

sorry I didn't get to this earlier, wish I could mark issues as unread again..

You might want to look at [PCM](https://en.wikipedia.org/wiki/Pulse-code_modulation#Modulation) which is the only modulation I cared about until now.

In PCM you actually deal with discrete points. Lets say you generate a sine wave and save it as pcm wav. Plotting that signal with the int value of each sample gets you an equivalent picture to what audacity would show you when opening that wav file.

So in short: Yes, you want the `ReadSample()` method and then do some fancy processing to compare the values to get your definition of 'audio level' :smile: 

For a simple sine wave this just might be running `max()` over all the ints but there are much more sophisitacted methods of course.

The `GetDumbReader()` was added to pipe processing to external programs that do the decoding from a slice of bytes to int by themselves.


Hope this helps.
